### ISSUE TITLE
[integration] Integrating multiple AI models

### ISSUE THREAD

**Issue by leeyun** [boiler-efficiency-factors/boiler-efficiency-factors#24]
- [ ] Logistic Regressions
- [ ] Gradient Boosting
- [ ] Random Forest
- [ ] XGBoost
- [ ] lightGBM
